src/py42/sdk/queries/fileevents/filters/event_filter.py

```diff
@@ -46,4 +86,43 @@ class InsertionTimestamp(QueryFilterTimestampField):
 
 
 class Source(FileEventFilterStringField):
+    """Class that filters events by event source.
+
+    Available event types are provided as class attributes:
+        - :attr:`Source.ENDPOINT`
+        - :attr:`Source.GOOGLE_DRIVE`
+        - :attr:`Source.ONE_DRIVE`
+        - :attr:`Source.BOX`
+        - :attr:`Source.GMAIL`
+        - :attr:`Source.OFFICE_365`
+
+    Example::
+
+        filter = Source.isin([Source.ENDPOINT, Source.BOX])
+
+    """
+
     _term = u"source"
+
+    ENDPOINT = u"Endpoint"
+    GOOGLE_DRIVE = u"GoogleDrive"
+    ONE_DRIVE = u"OneDrive"
+    BOX = u"Box"
+    GMAIL = u"Gmail"
+    OFFICE_365 = u"Office365"
+
+    @staticmethod
+    def choices():
+        return filter_attributes(Source)
+
+
+class RiskIndicatorMimeType(QueryFilterBooleanField):
+    """Class that filters events if file type mismatch the file's mime type."""
+
+    _term = u"mimeTypeMismatch"
+
+
+class RiskIndicatorActiveHours(QueryFilterBooleanField):
+    """Class that filters events by events occurred outside active hours."""
```

Comment:
`...filters events by whether or not they occurred outside a user's typical working hours`.

Reply:
Changed.